### ISSUE TITLE
Kaspersky no me deja minar / Kaspersky doesn't let me undermine

### ISSUE THREAD

**Issue by Kanopola** [vertcoin-project/one-click-miner-vnext#583]
![Screenshot_2](https://github.com/vertcoin-project/one-click-miner-vnext/assets/56226178/103d9918-a0f4-4963-aafa-757c4d585050)

Espero que el programa mine de forma normal. / I hope the Mine program normally.

Captura adjunta. / Attached capture.

Cuando está verficando y creando el archivo de datos da error porque dice que debo excluir el programa "VertaHashMiner.exe" del antivirus. / When you are verifining and creating the data file, it gives error because it says that I must exclude the program "vertahashminer.exe" of antivirus.

2.2 beta1-cda2154. Lo obtuve en la Web Oficial: https://vertcoin.org/ // I get it on the official website: https://vertcoin.org/

Estoy usando una Laptop Dell Precision M4700 con la tarjeta de video externa Nvidia Quadro K2000M en Windows 10 64 bits. / I am using a M4700 precision laptop with the Nvidia Quadro K2000M external video card on Windows 10 64 bits.

Estuve usando la versión 1.0 sin problemas y tenía minado 18 millones de Satoshis. Pero, desde que empecé a usar la versión 2.2 empezaron los problemas; y ahora tampoco puedo usar la versión anterior. / I was using version 1.0 without problems and had 18 million satoshis. But, since I started using version 2.2 the problems began; And now I can't use the previous version.

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

Por eso momento es todo lo que tengo y espero que me puedan ayudar con alguna solución lo antes posible.

Nota: estaré probando una posible solución temporal usando windows 7 en una máquina virtual como Virtualbox, para ver si así logro evitar que algún antivirus lo bloquee. También espero poder usar el archivo DATA de la versión anterior para recuperar lo que tengo minado.

That's why moment is all I have and I hope you can help me with some solution as soon as possible.

Note: I will be testing a possible temporary solution using Windows 7 on a virtual machine like Virtualbox, to see if I can prevent some antivirus from blocking it. I also hope to use the data data from the previous version to recover what I have mined.

¡Muchas gracias de antemano! ¡Saludos!

Thank you very much in advance! Greetings!

**Comment by KforG** [comment]
The only real solution to this is to properly exclude the miner from the antivirus (Including Windows Defender), some may be more unwilling to let you do this than others. The miner is located in the data directory of OCM which has the file path `%appdata%\vertcoin-ocm` on windows.

If this doesn't work make sure you backup the keyfile.hex located in the above mentioned data directory. This file contains the private key of your mined coins, thus the file contains your coins.

You can try deleting the entire data directory (except the keyfile) and resume using version 1.0.

If none of the above solutions fixes your problem, you can try to extract the private key from the keyfile: https://github.com/KforG/keyfile-generator-extractor
Remember to do your own research and not just trust me blindly. As always I would suggest you build this program yourself and don't use the compiled binaries to eliminate any potential vulnerabilities. Join the Vertcoin discord if you have any issues with this.

**Comment by Kanopola** [comment]
> The only real solution to this is to properly exclude the miner from the antivirus (Including Windows Defender), some may be more unwilling to let you do this than others. The miner is located in the data directory of OCM which has the file path `%appdata%\vertcoin-ocm` on windows.
> 
> If this doesn't work make sure you backup the keyfile.hex located in the above mentioned data directory. This file contains the private key of your mined coins, thus the file contains your coins.
> 
> You can try deleting the entire data directory (except the keyfile) and resume using version 1.0.
> 
> If none of the above solutions fixes your problem, you can try to extract the private key from the keyfile: https://github.com/KforG/keyfile-generator-extractor Remember to do your own research and not just trust me blindly. As always I would suggest you build this program yourself and don't use the compiled binaries to eliminate any potential vulnerabilities. Join the Vertcoin discord if you have any issues with this.

Well, the only viable solution I found was to format the computer. After I installed the Kaspersky antivirus again, it did not block the program in version 2.2, but now it only notifies me.

I also added exclusions to the program, but it just keeps notifying me, when it shouldn't even do that anymore.

Thanks for the suggestions; Since, of them I took the one that I wanted to know about where the balance of the wallet is saved, since, as I did not know, I had made backups of the entire contents of the folder, but now I know what I should save as a priority.

Luckily, when I installed version 2.2 and replaced the files I had from 1.0, everything was normal, but only the file containing the key was needed.

Thank you very much and I hope this helps others too.

Greetings!

Bueno, la única solución viable que encontré fue formatear la computadora. Luego que instalé el antivirus Kaspersky de nuevo no me bloquea el programa en su versión 2.2, sino que, ahora solo me notifica.

También agregué exclusiones al programa, pero solo me sigue notificando, cuando ya ni eso debería.

Gracias por las sugerencias; ya que, de ellas tomé la que quería saber sobre dónde se guarda el balance de la billetera, ya que, como no sabía, había hecho unos respaldos del contenido de la carpeta completo, pero ahora sé lo que sí debo guardar con prioridad.

Por suerte, cuando instalé la versión 2.2 y reemplacé los archivos que tenía de la 1.0, todo quedó normal, pero solo era necesario el archivo que contiene la clave.

Muchas gracias y espero que esto le sirva a otros también.

¡Saludos!
